block/manager_test.go

```diff
@@ -94,3 +98,94 @@ func TestIsDAIncluded(t *testing.T) {
 	m.blockCache.setDAIncluded(hash.String())
 	require.True(m.IsDAIncluded(hash))
 }
+
+func TestSubmitBlocks(t *testing.T) {
+	require := require.New(t)
+	ctx := context.Background()
+	logger := test.NewFileLoggerCustom(t, test.TempLogFileName(t, t.Name()))
+
+	// Create a minimalistic block manager
+	m := &Manager{
```

Comment:
[non blocking] can be done in a follow up, but I don't think we have a helper yet to create a manager in testing in the block package, so it might be helpful to pull this out to help with testing other areas.